src/Shamsheer.Service/Helpers/CustomAttributes/EmailValidationAttribute.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Net.Mail;
+using System.Text.RegularExpressions;
+using System.ComponentModel.DataAnnotations;
+
+[AttributeUsage(AttributeTargets.Property, AllowMultiple = false)]
+public class EmailValidationAttribute : ValidationAttribute
```

Comment:
Dont use` Validation` suffix to name attribute in this case! It may be `ShEmail` (Shamsheer email)